### ISSUE TITLE
Vendor-Prefixed unitless values get 'px' appended

### ISSUE THREAD

**Issue by namuol** [blakeembrey/free-style#2]
Example style:

```js
{
  flexGrow: 2,
  WebkitFlexGrow: 2,
}
```

Produces this style:

```css
.foo {
  flex-grow: 2;
  -webkit-flex-grow: 2px;
}
```

There's no real workaround, either. I've tried changing my code to this:

```js
{
  flexGrow: '2',
  WebkitFlexGrow: '2',
}
```

...but it still treats the `'2'` as a number and appends the `'px'` unit.

Am I missing something?

**Comment by blakeembrey** [comment]
Nope, not missing anything. It wasn't on the number whitelist. We can add it to the list: https://github.com/blakeembrey/free-style/blob/master/src/free-style.ts#L15-L31. Otherwise, we can append `px` to only numbers not on here? Or we can remove this "feature" all together to avoid future issues, but you'll have to start adding `px` now. Any preference?

**Comment by blakeembrey** [comment]
Released with `free-style@0.5.3`.
